service/http.directory/src/test/java/org/kaazing/gateway/service/http/directory/HttpDirectoryServiceIT.java

```diff
@@ -134,223 +147,240 @@ public void testTcpConnectAndClose() throws Exception {
-    @Test(timeout = 5000)
+    @Test()
     public void testEmptyHostHeaderWithRelativeUri() throws Exception {
         robot.finish();
     }
 
     @Specification("host/host.header.invalid")
-    @Test(timeout = 5000)
+    @Test()
     public void testInvalidHostHeader() throws Exception {
         robot.finish();
     }
 
     @Specification("host/host.header.not.present")
-    @Test(timeout = 5000)
+    @Test()
     public void testAbsentHostHeader() throws Exception {
         robot.finish();
     }
 
     @Specification("host/host.header.port.not.set")
-    @Test(timeout = 5000)
+    @Test()
     public void testAbsentPortInHostHeader() throws Exception {
         robot.finish();
     }
 
     @Specification("host/host.valid.header.with.absolute.uri")
-    @Test(timeout = 5000)
+    @Test()
     public void testValidHostHeaderWithAbsoluteUri() throws Exception {
         robot.finish();
     }
 
     @Specification("host/host.no.header.with.absolute.uri")
-    @Test(timeout = 5000)
+    @Test()
     public void testNoHostHeaderWithAbsoluteUri() throws Exception {
         robot.finish();
     }
 
     // ///////////////////// METHOD //////////////////////
     @Specification("method/method.connect")
-    @Test(timeout = 5000)
+    @Test()
     public void testConnectMethod() throws Exception {
         robot.finish();
     }
 
     @Specification("method/method.delete")
-    @Test(timeout = 5000)
+    @Test()
     public void testDeleteMethod() throws Exception {
         robot.finish();
     }
 
     @Specification("method/method.head")
-    @Test(timeout = 5000)
+    @Test()
     public void testHeadMethod() throws Exception {
         robot.finish();
     }
 
     @Specification("method/method.options")
-    @Test(timeout = 5000)
+    @Test()
     public void testOptionsMethod() throws Exception {
         robot.finish();
     }
 
     @Specification("method/method.bogus")
-    @Test(timeout = 5000)
+    @Test()
     public void testBogusMethod() throws Exception {
         robot.finish();
     }
 
     @Specification("method/method.post")
-    @Test(timeout = 5000)
+    @Test()
     public void testPostMethod() throws Exception {
         robot.finish();
     }
 
     // //////////////// URI ////////////////////////
     @Specification("uri.hex.encoded")
-    @Test(timeout = 5000)
+    @Test()
     public void testHexEncodedUri() throws Exception {
         robot.finish();
     }
 
-    @Specification("uri.too.long")
-    @Test(timeout = 5000)
-    public void testUriTooLong() throws Exception {
-        robot.finish();
-    }
-
     @Specification("invalid.uri.with.space")
-    @Test(timeout = 5000)
+    @Test()
+    
     public void testInvalidUri() throws Exception {
         robot.finish();
     }
 
     @Specification("uri.with.params")
-    @Test(timeout = 5000)
+    @Test()
     public void testUriWithParams() throws Exception {
         robot.finish();
     }
+    
+    @Specification("get.file.directory")
+    @Test()
```

Comment:
recommend changing script name to match test name with dots, i.e. get.hello.world.file